app/controllers/application_controller.rb

```diff
@@ -3,4 +3,16 @@
 class ApplicationController < ActionController::API
   include Localization
   include ErrorHandleable
+
+  rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
+
+  def record_not_found(exception)
+    message = if exception.model == 'SearchResult'
```

Comment:
Would this be more type-safe?

```suggestion
    message = if exception.model.is_a?(SearchResult)
```

Reply:
That implementation should be revised entirely as say, we add more models, will we have more conditions 🤯 Rails has a method to humanize resources name. Also, we usually use localized messages with a placeholder (see https://github.com/nimblehq/ewa-payroll-web/blob/develop/config/locales/activerecord.en.yml#L983)